js/site-search.js

```diff
@@ -44,8 +48,19 @@ bcpl.siteSearch = (($, window, constants) => {
 		const searchTerms = getSearchTerms();
 
 		if (searchTerms.length) {
+			const baseCatalogUrl = constants.baseCatalogUrl;
 			const searchUrl = constants.search.urls.catalog;
-			activeWindow.location.href = `${searchUrl}${searchTerms}`; // eslint-disable-line 			
+			activeWindow.location.href = `${baseCatalogUrl}${searchUrl}${searchTerms}`; // eslint-disable-line 			
+		}
+	};
+
+	const searchEvents = (activeWindow) => {
+		const searchTerms = getSearchTerms();
+
+		if (searchTerms.length) {
+			const baseWebsiteUrl = constants.baseWebsiteUrl;
+			const searchUrl = constants.search.urls.events;
+			activeWindow.location.href = `${baseWebsiteUrl}${searchUrl}${searchTerms}`; // eslint-disable-line 			
```

Comment:
in this case, search terms contains the ? for query params?

Reply:
Correct.